index.bs

```diff
@@ -722,6 +809,11 @@ Example of declaring an iframe that is allowed to access VR features.
 <pre class="idl">
 partial interface Gamepad {
   readonly attribute unsigned long displayId;
+  
+  readonly attribute boolean hasPosition;
+  readonly attribute boolean hasOrientation;
+  
+  boolean getPose(VRCoordinateSystem coordinateSystem, VRPose pose);
```

Comment:
Okay, so here's the big issue for Gamepad that I still don't have a good answer for: A gamepad may have a pose and not be related to a `VRDisplay` at all. This covers basically anything with a gyro in it (Like the PS4 or Wii controllers). They'll need a way to report values in their own CoordinateSystem, which will likely look like an Attached one but it's really hard to say.

Like I said, I don't have a good answer for this. Suggestions welcome!